e621dl.py

```diff
@@ -2,10 +2,16 @@
 
 import os
 import httpx
+import logging
 from lib import constants
 from lib import local
 from lib import remote
 
+LOG_FILENAME = 'ErrorLog.txt'
+logging.basicConfig(filename=LOG_FILENAME, level=logging.ERROR, format="%(asctime)s %(message)s")
+CheckLogs = 'FALSE'
```

Comment:
Use boolean instead of string, follow naming convention for variables.
`checkLogs = False`